src/components/Button/Button.module.scss

```diff
@@ -180,14 +180,18 @@
 
     &.disabled {
       @include inner-border(transparent);
-      background: var(--color-background-muted-alpha);
       color: var(--color-text-muted);
 
       .iconWrapper {
         color: var(--color-text-muted);
       }
     }
   }
+
+  &.disabled {
+    cursor: not-allowed;
```

Comment:
I'm also not the greatest fan of this cursor. Not sure how strong do you feel about it 😄
I won't mind it if you really feel like though.

Reply:
My thinking here was that since the difference in contrast between default and disabled buttons is so low, it would make sense to have that additional layer of visual indication that a given button is not active _by design_, and not due to, say, a technical error. Also, [this great read](https://www.smashingmagazine.com/2021/08/frustrating-design-patterns-disabled-buttons/#:~:text=change%20the%20cursor,)%2C) gave me a bit more confidence in adding this line of code. 😅

So yeah, I feel pretty strong about it atm. :D:D